shell_integration/windows/NCContextMenu/CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ set_target_properties(NCContextMenu
             ${BIN_OUTPUT_DIRECTORY}
         RUNTIME_OUTPUT_DIRECTORY
             ${BIN_OUTPUT_DIRECTORY}
+        VERSION
+            ${WINDOWS_INTEGRATION_VERSION}
+        SOVERSION
+            ${WINDOWS_INTEGRATION_SOVERSION}
```

Comment:
guess these are no longer required?  (same for the other _CMakeLists.txt_)

Reply:
removed